code/modules/antagonists/infiltrator/infiltrator.dm

```diff
@@ -0,0 +1,89 @@
+/datum/antagonist/infiltrator
+	name = "Syndicate Infiltrator"
+	roundend_category = "syndicate infiltrators" //just in case
+	antagpanel_category = "Infiltrator"
+	job_rank = ROLE_INFILTRATOR
+	var/datum/team/infiltrator/infiltrator_team
+	var/always_new_team = FALSE //If not assigned a team by default ops will try to join existing ones, set this to TRUE to always create new team.
+	var/send_to_spawnpoint = TRUE //Should the user be moved to default spawnpoint.
+
+/datum/antagonist/infiltrator/proc/update_synd_icons_added(mob/living/M)
+	var/datum/atom_hud/antag/sithud = GLOB.huds[ANTAG_HUD_INFILTRATOR]
+	sithud.join_hud(M)
+	set_antag_hud(M, "synd")
+
+/datum/antagonist/infiltrator/proc/update_synd_icons_removed(mob/living/M)
+	var/datum/atom_hud/antag/sithud = GLOB.huds[ANTAG_HUD_INFILTRATOR]
+	sithud.leave_hud(M)
+	set_antag_hud(M, null)
+
+/datum/antagonist/infiltrator/apply_innate_effects(mob/living/mob_override)
+	var/mob/living/M = mob_override || owner.current
+	update_synd_icons_added(M)
+
+/datum/antagonist/infiltrator/remove_innate_effects(mob/living/mob_override)
+	var/mob/living/M = mob_override || owner.current
+	update_synd_icons_removed(M)
+
+/datum/antagonist/infiltrator/greet()
+	owner.current.playsound_local(get_turf(owner.current), 'sound/ambience/antag/ops.ogg',100,0)
+	to_chat(owner, "<span class='userdanger'>You are a syndicate infiltrator!</span>")
+	to_chat(owner, "<span class='notice bold'>Your job is to infiltrate [station_name()], and complete our objectives.</span>")
+	to_chat(owner, "<span class='notice'>You have an uplink implant, precharged with 30 TC. Use it wisely.</span>")
+	to_chat(owner, "<span class='notice'>You also have an internal radio, for communicating with your team-mates at all times.</span>")
+	to_chat(owner, "<font size=2><span class='notice bold'>Do NOT kill or destroy needlessly, as this defeats the purpose of an 'infiltration'!</span></font>")
```

Comment:
don't forget to make a jobban type for this if you want this to be enforced at all.

Reply:
added to jobban panel.